src/dmd/lexer.d

```diff
@@ -536,9 +536,6 @@ class Lexer
                     anyToken = 1;
                     if (*t.ptr == '_') // if special identifier token
                     {
-                        // Lazy initialization
-                        TimeStampInfo.initialize(t.loc);
```

Comment:
I can't believe someone decided it was a good idea to put initialization of global data nested so deeply... quite the concurrentcy pitfall :D

Reply:
👀

Reply:
`git blame` tells me the code was like that from the prehistoric times (i.e. before ddmd), so you're spared from the wrath of the gods of code 😄